requirements/ci.txt

```diff
@@ -6,11 +6,11 @@
 #
 certifi==2022.6.15
     # via requests
-charset-normalizer==2.0.12
+charset-normalizer==2.1.0
```

Comment:
I actually can't resolve dependencies easily without doing an upgrade, as it turns out. If I revert the compiled requirements, remove `--upgrade` from the Makefile, and run `make upgrade`, then I get this error:

```
pip-compile  -o requirements/dev.txt requirements/dev.in
Could not find a version that matches charset-normalizer==2.0.12,==2.1.0 (from -r requirements/ci.txt (line 9))
Tried: 0.1.7, 0.1.8, 0.2.0, 0.2.1, 0.2.2, 0.2.3, 0.3.0, 1.0.0, 1.1.0, 1.1.1, 1.2.0, 1.3.0, 1.3.1, 1.3.2, 1.3.3, 1.3.4, 1.3.5, 1.3.5, 1.3.6, 1.3.6, 1.3.7, 1.3.7, 1.3.8, 1.3.8, 1.3.9, 1.3.9, 1.4.0, 1.4.0, 1.4.1, 1.4.1, 2.0.0, 2.0.0, 2.0.1, 2.0.1, 2.0.2, 2.0.2, 2.0.3, 2.0.3, 2.0.4, 2.0.4, 2.0.5, 2.0.5, 2.0.6, 2.0.6, 2.0.7, 2.0.7, 2.0.8, 2.0.8, 2.0.9, 2.0.9, 2.0.10, 2.0.10, 2.0.11, 2.0.11, 2.0.12, 2.0.12, 2.1.0, 2.1.0
Skipped pre-versions: 0.1a0, 0.1.1a0, 0.1.2b0, 0.1.4b0, 0.1.5b0
There are incompatible versions in the resolved dependencies:
  charset-normalizer==2.1.0 (from -r requirements/quality.txt (line 32))
  charset-normalizer==2.0.12 (from -r requirements/ci.txt (line 9))
```

That's because quality.txt and ci.txt are compiled separately and then combined in dev.txt. I could manually fix it, but it might be easier to just run a normal `make upgrade` here.

Reply:
Sure, I'm down with make upgrade for now. in the future we should be more careful, but bias towards action etc.